src/pages/Review.less

```diff
@@ -62,11 +62,10 @@
         position: relative;
         flex: 0 0 auto;
         display: block;
-        border-top: 1px solid @gray;
-        border-bottom: 1px solid @gray;
-        margin-bottom: 10px;
-        color: @gray;
+        color: @brand-info;
         cursor: row-resize;
+        background: @gray-darker;
+        padding: 2px 0;
```

Comment:
I'm pretty sure you didn't push all changes, please check it. This lines can't produce the result we see on the screenshot.

Reply:
I've pushed all the changes @smacker

Reply:
Nope. It's what I see if I checkout to this branch:
<img width="1322" alt="screen shot 2018-02-19 at 20 02 30" src="https://user-images.githubusercontent.com/406916/36393441-e26f5b1c-15af-11e8-8eb4-7496e584a196.png">

Reply:
@smacker I beg your pardon. I've just pushed what was supposed to be pushed. Now it's all ok for the merge.

![image](https://user-images.githubusercontent.com/1746146/36393926-a9c62ae4-15a9-11e8-9bbd-4592b8f76af2.png)